components/atom/radioButton/src/index.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-AtomRadioButton'
+const CLASS_HIDDEN = `is-hidden`
+
+const ERROR_STATES = {
```

Comment:
I get the feeling of how you use within the components this object is more like response states. If so, I think that for greater readability it should be `RESPONSE_STATUS`

Reply:
Removed ERROR_STATES logic as this component doesn't really need to be aware of this (the specification doesn't reflect any behaviour in this level regarding errors)